src/main/java/input/HandleInput.java

```diff
@@ -0,0 +1,164 @@
+package input;
+
+import exceptions.EmptyTaskException;
+import exceptions.MissingDeadlineException;
+import exceptions.MissingStartException;
+import exceptions.UnknownInputException;
+import tasks.Deadline;
+import tasks.Event;
+import tasks.Task;
+import tasks.ToDo;
+
+import java.util.Scanner;
+
+public class HandleInput {
+    static String line = "\t____________________________________________________________";
+    static Task[] tasks = new Task[100];
+    static int count = 0;
+
+    public static void echo(String input) {
+        System.out.println("\t Got it. I've added this task: ");
+        System.out.println(("\t   " + tasks[count - 1].toString()));
+        String task = (count == 1) ? "task " : "tasks ";
+        System.out.println("\t Now you have " + count + " " + task + "in the tasks.");
+    }
+
+    public static void sayBye() {
+        System.out.println("\t Bye. Hope to see you again soon!");
+    }
+
+    public static void createTodo(String input) throws EmptyTaskException {
+        String todo = input.substring(input.indexOf("todo ") + 5);
+        if (todo.isBlank()) {
+            throw new EmptyTaskException();
+        }
+        tasks[count] = new ToDo(todo);
+        count++;
+        echo(input);
+    }
+
+
+    public static void createDeadline(String input) throws EmptyTaskException, MissingDeadlineException {
+        String description = input.replaceFirst("deadline ", "");
+        int by = description.indexOf("/");
+        if (by == -1){
+            throw new MissingDeadlineException();
+        }
+        String deadline = description.substring(0, by - 1);
+        if (deadline.isBlank()){
+            throw new EmptyTaskException();
+        }
+        String date = description.substring(by + 4);
+        tasks[count] = new Deadline(deadline, date);
+        count++;
+        echo(input);
+    }
+
+    public static void createEvent(String input) throws EmptyTaskException,
+            MissingDeadlineException, MissingStartException {
+        String description = input.replaceFirst("event ", "");
+        int from = description.indexOf("/from");
+        if (from == -1) {
+            throw new MissingStartException();
+        }
+        int by = description.indexOf("/to");
+        if (by == -1) {
+            throw new MissingDeadlineException();
+        }
+        String startDate = description.substring(from + 6, by - 1);
+        String endDate = description.substring(by + 4);
+        if (from == 0) {
+            throw new EmptyTaskException();
+        }
+        String event = description.substring(0, from - 1);
```

Comment:
Avoid using magic literals. It is unclear to the reader what the numbers are supposed to mean. This also applies to string literals!